FrontMen/fm-hours
# Feat: work schema api

# Changes

## Related issues

Resolves #42 

## Added

- Work-scheme types
- Helper function to mark day as holiday based on work-scheme
- `isLeaveDay` added to `WeekDay` type
- Show indicators and Frey out leave days

## Removed

- Functions that automatically creates projects based on work-scheme results

## Changed

- No longer mocked work-scheme
- App also checks work-scheme to see if is holiday

## How to test

- Check if the holidays are correctly displayed
- Check if leave days are correctly displayed

## Screenshots

N/A
